homeassistant/components/bluetooth_tracker/device_tracker.py

```diff
@@ -61,11 +68,15 @@ def discover_devices(device_id: int) -> list[tuple[str, str]]:
         device_id=device_id,
     )
     _LOGGER.debug("Bluetooth devices discovered = %d", len(result))
-    return result
+    return result  # type: ignore[no-any-return]
 
 
 async def see_device(
-    hass: HomeAssistant, async_see, mac: str, device_name: str, rssi=None
+    hass: HomeAssistant,
+    async_see: Callable,
```

Comment:
You didn't update this one.
```suggestion
    async_see: Callable[..., Awaitable[None]],
```

Reply:
damn :facepalm: :smile: